src/pieces/commandManager.ts

```diff
@@ -85,6 +85,10 @@ async function runCommand(msg: Message) {
 
 	const command = getCommand(msg.client, commandName);
 	if (!command || command.enabled === false) return;
+	if (msg.channel.id !== CHANNELS.SAGE && command.restricted) {
+		msg.author.send('This command is only available in #sages_place.');
```

Comment:
Also this won't notify the person if they have DMs disabled from server members (it also errors)

Reply:
This. Might be best to make it an ephemeral response once slash commands are fully implemented.

Reply:
Very good points, thanks guys